src/vm/arm64/gmscpu.h

```diff
@@ -29,6 +29,13 @@ struct MachState {
     
     BOOL   isValid()    { LIMITED_METHOD_DAC_CONTRACT; return _isValid; }
     TADDR  GetRetAddr() { LIMITED_METHOD_DAC_CONTRACT; return _pc; }
+
+#ifdef FEATURE_PAL
+    // On PAL, we don't always have the context pointers available due to
+    // a limitation of an unwinding library. In such case, preserve
+    // the unwound values.
+    CalleeSavedRegisters m_Unwound;
```

Comment:
This is not needed for ARM64. The reason we have it for AMD64 is OSX which uses a different libunwind (the one that's part of the LLVM) that doesn't support `unw_get_save_loc`. See https://github.com/dotnet/coreclr/pull/527/files.